### DISCUSSION TITLE
VR control of head / arms + Inverse Kinematics

### DISCUSSION THREAD

**Discussion by c4b4d4** [pixiv/three-vrm#1145] [Help]
Has anyone tried to control a character in first-person view using **Web VR** controls?

I’ve been able to control floating hands before, but I want to start controlling full avatars (hands/arms/head).

How would you control the arms of the avatar using **Inverse Kinematics**, knowing the **world position** of the user’s hands + head.

Any ideas?

Thanks in advance.

**Comment by 0b5vr** [comment]
We don't provide any Humanoid IK systems for now.

However, there is a case that uses IK with three-vrm in the community: https://github.com/exokitxr/avatars

**Comment by xaxfan** [comment]
来信已收到，谢谢！
